### PR TITLE
Ruchi's Data Exploration

### PR DESCRIPTION
**Pull request by ruchiasthana**
In my data exploration I include:

Data exploration for IMS Data and Parsing logic of asc files
Exploration of LSTM and application of LSTM to IMS Data

### CODE REVIEW

**Review by JuLieAlgebra — APPROVED** [review]
Awesome!! Thanks Ruchi

File: workspaces/ruchi/example.py
Comment (SooluThomas) [inline workspaces/ruchi/example.py:1]: I guess you can remove this file?

**Review by SooluThomas — APPROVED** [review]
This is awesome! I love the explanation provided!! Just one non-blocking comment.

File: workspaces/ruchi/example.py
Comment (JuLieAlgebra) [reply workspaces/ruchi/example.py:1]: @SooluThomas would you mind if I marked this as resolved? I'd love to get our last open PR closed out before meeting on sunday night :)

**Pull request merged**